arbitrum-docs/node-running/how-tos/running-a-feed-relay.mdx

```diff
@@ -12,4 +16,4 @@ When running more than one node, you want to run a single feed relay per datacen
   ```
 
 
-Note that Arbitrum classic does not communicate with Nitro sequencer, so classic relay is no longer used.
\ No newline at end of file
+⚠️ Note that Arbitrum classic does not communicate with Nitro sequencer, so classic relay is no longer used.
```

Comment:
Minor detail: this doesn't sound like a warning, but more like a notice. Should we use the speaker here, or the "i" for information icon, or no icon?

Reply:
I'm ok with having no icon since it's just an info. Sounds good?